### DISCUSSION TITLE
Questions about experimental features

### DISCUSSION THREAD

**Discussion by andyluooo** [microsoft/pylance-release#4413] [Q&A] [answered]
I've just found the experimental options `Pylance Lsp Client Enabled` and `Enable Sync Server`, but don't know what are they for. Could anyone explain a bit for me? Great thanks.

**Comment by debonte** [comment]
@rchiodo, @heejaechang

**Comment by heejaechang** [comment, marked as answer]
those are not meant to be changed by users. those options might not stay based on on-going experiment result. that's why there is no explanation in `README`

that said, both of them are testing some internal architecture changes, there should be no UX changes.

**Reply by andyluooo** [reply]
> those are not meant to be changed by users. those options might not stay based on on-going experiment result. that's why there is no explanation in `README`
> 
> that said, both of them are testing some internal architecture changes, there should be no UX changes.

Thanks.
